gfx/NeoDays/pngs_tiles_10x10/items/tools/lighting/atomic_light.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "atomic_light"
+  ],
+  "fg": "atomic_light",
+  "//": "due to a vehicle part json referring to this sprite, this file is necessary for the item to display"
```

Comment:
same say say what part, or combine the two json files in one by putting the item and the vehicle part in the id field

Reply:
Done; mentioned the vehicle part file.